CHANGELOG.md

```diff
@@ -94,6 +94,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 - Replace jboss-annotations-api_1.2_spec with jakarta.annotation-api ([#7836](https://github.com/opensearch-project/OpenSearch/pull/7836))
 - Add min, max, average and thread info to resource stats in tasks API ([#7673](https://github.com/opensearch-project/OpenSearch/pull/7673))
+- MasterService Debug logs for computation time to use system clock time instead of cached time ([#7902](https://github.com/opensearch-project/OpenSearch/pull/7902))
```

Comment:
I'd rephrase this as "Compute accurate elapsed time in MasterService logging output"

Reply:
It's really a fix, no? In which case it should be "Correct system clock time in MasterService debug output" and go into fixes.

Reply:
"cached" time was used as an optimization but resulted in 200ms precision/granularity in the elapsed time reported in the logs. One could argue that was a deliberate tradeoff, but calling it a fix is probably simpler.